Hunted Mobile/Hunted Mobile/Hunted Mobile/Model/Map.cs

```diff
@@ -1,69 +1,66 @@
 ﻿using Hunted_Mobile.Model.GameModels;
 
-using Mapsui.Geometries;
-using Mapsui.UI.Forms;
-using Mapsui.UI.Objects;
-
 using System;
 using System.Collections.Generic;
+using System.Collections.ObjectModel;
 using System.Linq;
 
 namespace Hunted_Mobile.Model {
     public class Map {
-        private List<Player> users = new List<Player>();
-
-        private List<Loot> loot = new List<Loot>();
-
-        public Player PlayingUser { get; set; }
-        public Boundary GameBoundary { get; set; }
+        private readonly ObservableCollection<Player> players = new ObservableCollection<Player>();
+        private readonly ObservableCollection<Loot> loot = new ObservableCollection<Loot>();
 
-        public List<Police> Police => users.Where(user => user is Police).Select(user => new Police(user)).ToList();
-        public List<Thief> Thiefs => users.Where(user => user is Thief).Select(user => new Thief(user)).ToList();
+        public ICollection<Player> Players {
+            get => players;
+            set {
+                players.Clear();
```

Comment:
Kan `players = value;` niet? Anders is het ook gewoon goed

Reply:
Dan wordt de CollectionChanged event van de ObservableCollection niet aangeroepen, dus doet ie geen validatie check.